test/Microsoft.ML.Tests/Scenarios/Api/TestApi.cs

```diff
@@ -294,7 +294,7 @@ private List<BreastCancerExample> ReadBreastCancerExamples()
         public void TestTrainTestSplit()
         {
             var mlContext = new MLContext(0);
-            
+
             var dataPath = GetDataPath("adult.tiny.with-schema.txt");
```

Comment:
one

Reply:
one what?

---
In reply to: [263175595](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263175595) [](ancestors = 263175595)

Reply:
one empty line, or did you mean to have two?

---
In reply to: [263510301](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263510301) [](ancestors = 263510301,263175595)

Reply:
It's actually codeflow specific. it show two new lines if you remove leading spaces from it

---
In reply to: [263685676](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263685676) [](ancestors = 263685676,263510301,263175595)